app/models/form_handler.rb

```diff
@@ -14,9 +14,12 @@ def current_form
     forms[forms.keys.max_by(&:to_i)]
   end
 
-private
+  def sales_forms
+    sales_sections = [Form::Sales::Setup::Sections::Setup]
+    { "2022_2023_sales" => Form.new(nil, "2022_2023_sales", sales_sections, "sales") }
```

Comment:
Do we need to name these? Could we get away with something we don't need to change every year, such as `current` and `next`?